java-plugin/pom.xml

```diff
@@ -16,21 +16,24 @@
     <url>https://github.com/green-code-initiative/ecoCode/tree/main/java-plugin</url>
 
     <dependencies>
+        <dependency>
```

Comment:
why do you add this dependency ?

Reply:
This is the package containing the rules specification documentations that are displayed in SonarQube when an issue is raised.